web/src/desktop_integration.js

```diff
@@ -63,3 +65,32 @@ if (window.electron_bridge !== undefined) {
         });
     });
 }
+
+export function initialize() {
+    if (window.electron_bridge === undefined) {
+        return;
+    }
+
+    $(document).on("click", "#open-self-hosted-billing", (event) => {
+        event.preventDefault();
+
+        const url = "/json/self-hosted-billing/";
+
+        channel.get({
+            url,
+            success(data) {
+                window.open(data.billing_access_url, "_blank");
```

Comment:
@andersk should this use the `noopener` and `noreferrer` options? https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noopener, like other new-tab opens from the web app?

Reply:
I think the answer is that it doesn't matter, in that we're opening something in a different browser. But I think we should specify these properties anyway assuming it still works if we do so, since it'd be unsafe to copy/paste this logic to be used within the normal web app. @mateuszmandera do you want to make that change?

Reply:
@timabbott Yup, still works